charts/everest/Makefile

```diff
@@ -8,7 +8,6 @@ prepare-chart: deps
 	yq eval -i '.olm.catalogSourceImage = "$(IMAGE_PREFIX)/everest-catalog"' values.yaml
 	yq eval -i '.operator.image = "$(IMAGE_PREFIX)/everest-operator"' values.yaml
 	yq eval -i '(.dependencies[] | select(.name == "everest-db-namespace")).version = "${VERSION}"' Chart.yaml
-	helm dependency update .
```

Comment:
Why did this get removed?

Reply:
This command already runs in `make deps`, so it is redundant here